org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "command": "qelectrotech",
+  "rename-desktop-file": "qelectrotech.desktop",
+  "rename-appdata-file": "qelectrotech.appdata.xml",
+  "rename-icon": "qelectrotech",
+  "copy-icon": true,
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=x11",
+    "--device=dri",
+    "--share=ipc",
+    "--filesystem=host"
+  ],
+  "modules": [
+    {
+      "name": "qelectrotech",
+      "buildsystem": "qmake",
+      "post-install": [
+        "mv /app/share/mime/packages/qelectrotech.xml /app/share/mime/packages/org.qelectrotech.QElectroTech.xml"
+      ],
+      "sources": [
+        {
+          "type": "dir",
+          "path": "../.."
```

Comment:
You need a tarball/git source here

Reply:
https://git.tuxfamily.org/qet/qet.git/tag/?id=0.8.0

Reply:
Or
https://github.com/qelectrotech/qelectrotech-source-mirror/releases/tag/0.8.0